syntax = "proto3";

package add_op;

service AddService {
  rpc AddRPC (AddRequest) returns (AddReply) {} 
}

message AddRequest {
  int32 a = 1;
  int32 b = 2;
}

message AddReply {
  int32 sum = 1;
}
